### PR TITLE
Added Remove button to store cart

### PR DESCRIPTION
**Pull request by RONNCC**
To address #134

### CODE REVIEW

File: app/views/store/items/_cart.html.erb [hunk 1]
```diff
@@ -6,6 +6,7 @@
       <th>Price</th>
       <th>Quantity</th>
       <th>Subtotal</th>
+      <th></th>
```
Comment (pkoenig10) [inline app/views/store/items/_cart.html.erb]: Title this column "Actions"

File: app/views/store/items/_items.html.erb [hunk 2]
```diff
@@ -5,6 +5,7 @@
       <th>Name</th>
       <th>Price</th>
       <th>Quantity Available</th>
+      <th></th>
```
Comment (pkoenig10) [inline app/views/store/items/_items.html.erb]: Title this column "Actions"

**Comment by purelogiq** [comment]
No other issues with this (besides existing issues with the store).

**Comment by RONNCC** [comment]
changes made according to review :)

**Comment by pkoenig10** [comment]
Two things:
1) The remove button should be red (btn-danger class).
2) The remove button and the add to cart button should be small (btn btn-xs class).

**Comment by RONNCC** [comment]
fixed

**Pull request merged**